xwiki-platform-core/xwiki-platform-oldcore/src/main/java/com/xpn/xwiki/objects/classes/ListClass.java

```diff
@@ -471,6 +504,9 @@ public BaseProperty newProperty()
             lprop = new DBStringListProperty();
         } else if (isMultiSelect()) {
             lprop = new StringListProperty();
+        // If we allow custom values, they might use a lot of space in DB, so better to put them in a LargeString.
```

Comment:
This comment does not seems to have anything if the following if, leftover ?

Reply:
right. I'll remove it.